### DISCUSSION TITLE
error: request for member 'm128_u32' in 'result',

### DISCUSSION THREAD

**Discussion by MunoDevelop** [NVIDIA-Omniverse/PhysX#253] [Q&A]
after build libs and includes, i built my project and it shows m128_u32 error
![스크린샷 2024-03-14 141325](https://github.com/NVIDIA-Omniverse/PhysX/assets/26921159/d41d920b-0828-43bf-b638-7f9006aa4cfe)
and cmake file like:

cmake_minimum_required(VERSION 3.7)

set(CMAKE_CXX_STANDARD 20)

project(CoreServer)

option(PHYSX_BUILD_TYPE debug)

# Include PhysX headers
include_directories("../../physx/include")

# Get all the .cpp files in the current directory and its subdirectories
#file(GLOB_RECURSE SOURCES "*.cpp" EXCLUDE "examples/*")

# Get all the header files in the current directory and its subdirectories
#file(GLOB_RECURSE HEADERS "*.h" "*.hpp" EXCLUDE "examples/*")

# Exclude CompilerId files
#list(REMOVE_ITEM SOURCES "${CMAKE_CURRENT_SOURCE_DIR}/CMakeFiles/${CMAKE_CXX_COMPILER_ID}/CMakeCXXCompilerId.cpp")
#list(REMOVE_ITEM HEADERS "${CMAKE_CURRENT_SOURCE_DIR}/CMakeFiles/${CMAKE_CXX_COMPILER_ID}/CMakeCXXCompilerId.h")

if(CMAKE_BUILD_TYPE AND CMAKE_BUILD_TYPE STREQUAL "Debug")
    message("Building snippet in debug configuration")
    add_compile_definitions(_DEBUG)
    link_directories("../../physx/bin/win.x86_64.vc142.mt/debug") # This is the path where PhysX libraries are installed
else()
    message("Building snippet in release configuration with PhysX ${PHYSX_BUILD_TYPE} configuration")
    add_compile_definitions(NDEBUG)
    link_directories("PhysX/bin/linux.clang/${PHYSX_BUILD_TYPE}") # This is the path where PhysX libraries are installed
endif()

add_executable(CoreServer main.cpp)

target_link_libraries(CoreServer
        PhysXExtensions_static_64.lib
        PhysX_static_64.lib
        PhysXPvdSDK_static_64.lib
        PhysXCommon_static_64.lib
        PhysXFoundation_static_64.lib

        pthread
        dl
)

**Comment by Niyll** [comment]
Run into the same problem today. I'm building project with clang-cl on windows, and it seems that I can't get access to m128_32 fields of VecU32V since they are declared as union members only in unix headers or msvc tools headers 
<img width="565" height="156" alt="image" src="https://github.com/user-attachments/assets/34fa8d52-0c09-4027-b7c2-5a5b8447a55c" />
Temporary solved this problem by #define PX_SIMD_DISABLED, but I'm not very happy with this hack
